src/features/MergeNetworks/utils/helper-functions.ts

```diff
@@ -0,0 +1,41 @@
+import { Pair } from "../models/DataInterfaceForMerge";
+
+//utility function to find index of a pair in a list
+export const findPairIndex = (pairs: Pair<string, string>[], uuid: string) => {
+    return pairs.findIndex(pair => pair[1] === uuid);
+};
+// Utility function for deep cloning objects that may contain arrays, maps, and functions
+export function deepClone<T>(obj: T): T {
```

Comment:
We use the package`lodash` which includes a deepClone function already.  Is there a specific case that this function you have here handles that lodash deepClone does not already handle?

Reply:
Okay, I will substitute the current deepClone with the one from `lodash`